src/vuex/modules/distribution.js

```diff
@@ -95,21 +97,28 @@ export default ({ node }) => {
       })
     },
     async withdrawAllRewards(
-      {
-        rootState: { session },
-        getters: { committedDelegations },
-        dispatch
-      },
-      { gas, gas_prices, password, submitType }
+      { rootState, getters, dispatch },
+      { gas, gasPrice, denom, validatorAddress, password, submitType }
     ) {
+      const top5Delegations = getTop5Delegations(getters.committedDelegations)
```

Comment:
should only limit to 5 delegations if the user is using the Ledger Nano S, but we can figure this out once we have more signers

Reply:
Indeed. There's some time to spare before the Nano X is in wide distribution, and we'll need to detect the difference. Is that right?

Reply:
yep